spec.yaml

```diff
@@ -89,10 +89,16 @@ paths:
                 "code": "RESOURCE_CREATED",
                 "message": "A new cluster has been created with ID 'wqtlq'"
               }
-        default:
-          description: Error
+        "401":
+          description: Permission Denied
```

Comment:
I would prefer `Permission denied` here (no title case), for consistency with `Cluster not found` etc.